activestorage/app/models/active_storage/blob.rb

```diff
@@ -101,7 +101,7 @@ def create_after_unfurling!(key: nil, io:, filename:, content_type: nil, metadat
     # be saved before the upload begins to prevent the upload clobbering another due to key collisions.
     # When providing a content type, pass <tt>identify: false</tt> to bypass
     # automatic content type inference.
-    def create_and_upload!(key: nil, io:, filename:, content_type: nil, metadata: nil, service_name: nil, identify: true, record: nil)
+    def create_and_upload!(io:, filename:, key: nil, content_type: nil, metadata: nil, service_name: nil, identify: true, record: nil)
       create_after_unfurling!(key: key, io: io, filename: filename, content_type: content_type, metadata: metadata, service_name: service_name, identify: identify).tap do |blob|
```

Comment:
It was originally intended to arrange the order of the keys, now the intent is broken by rubocop autocorrect.